packages/services/src/view-service/transaction-planner/index.ts

```diff
@@ -14,11 +14,13 @@ export const transactionPlanner: Impl['transactionPlanner'] = async (req, ctx) =
   const { indexedDb } = await services.getWalletServices();
 
   // Query IndexedDB directly to check for the existence of staking token
-  const nativeToken = await indexedDb.hasStakingAssetBalance(req.source);
+  const nativeToken = await indexedDb.hasStakingAssetBalance();
```

Comment:
What's the thinking around removing source here? Is the expectation that paying fees should not be scoped to one account and can be paid by any of your accounts? Is this consistent with the core repo? At the moment, in minifront, we've kinda been scoping all activity to a single account in a tx.

Reply:
I think the issue is that `source` should always be set, and we should not create transaction plans that are not linked to any account (address index)

Reply:
The fact that `hasStakingAssetBalance()` now doesn't take AddressIndex into account leads to a bug where the planner thinks he has a native token for the commission but can't spend it. This bug will be reproduced quite often